input/pagecontent/general-requirements.md

```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
 #### AU Core Profiles and Extensions
 The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU Core profiles and AU Core extensions defined for this implementation guide. An AU Core profile [StructureDefinitions](http://hl7.org/fhir/R4/structuredefinition.html) defines the minimum elements, extensions, vocabularies and value sets that **SHALL** be present and constrains the way elements are used when conforming to the profile.
```

Comment:
fix grammar - 'defines' should be 'define'

Reply:
No, it is correct in that 'defines' is current tense rather than 'will define' or future tense 'defined'.

Reply:
Yes. I know why I misread it - [StructureDefinitions](http://hl7.org/fhir/R4/structuredefinition.html) hyperlink text should be StructureDefinition.

Reply:
This is actually handled by a separate Jira - so we'll leave it that PR.